src/Common/Common.Api/Workspaces/WorkspacesClient.cs

```diff
@@ -60,5 +60,34 @@ public object NewWorkspaceAsUser(string workspaceName)
         {
             return (Workspace)this.Client.Groups.CreateGroup(new PowerBI.Api.V2.Models.GroupCreationRequest(name: workspaceName));
         }
+
+        public void MigrateWorkspaceCapacity(Guid workspaceId, Guid capacityId)
+        {
+            if (capacityId != Guid.Empty)
+            {
+                var request = new PowerBI.Api.V2.Models.AssignWorkspacesToCapacityRequest
+                {
+                    CapacityMigrationAssignments = new List<PowerBI.Api.V2.Models.CapacityMigrationAssignment>
+                    {
+                        new PowerBI.Api.V2.Models.CapacityMigrationAssignment
+                        {
+                            WorkspacesToAssign = new List<string> { workspaceId.ToString() },
```

Comment:
Minor: I'm not sure what the contract is but I see List<string> used here and for unassign it uses string array. If we can make it consistent that would be good but not a requirement

Reply:
Somehow I didn't get your comments and I merged it already. Let me open another PR to address this.